examples.ts

```diff
@@ -65,3 +66,24 @@ console.log({
   scenario: "diffusion_generation",
   price: getModelPricing(diffusion_generation_params),
 });
+
+// MRNA Diffusion Generation ----------------------------------------------------
+
+const mrnadiffusion_generation_params: MRNADiffusionGenerateParams = {
+  transform: Transforms.MRNA_DIFFUSION_GENERATE,
+  unmaskings_per_step: 3,
+  five_utr:
+    "ATGG<MASK><MASK><MASK><MASK>MKL<MASK><MASK><MASK><MASK><MASK><MASK><MASK>MKL",
```

Comment:
I don't know if this has to run but there are AA characters in this that will fail (MKL). This has to be A/T/G/C. Also not sure where this is hitting in the API: the ginkgo python AI client takes <mask> for consistency but the original model takes [MASK].

Reply:
ah good catch, I just copy-pasted something without checking, fixed now.

Re. your question, The pricing model is run on user requests before the request is sent to our servers. It doesn't fully validate sequences, the critical things we want from it is that it implements the right formula and parses correctly any request generated by the python client (hence <MASK> here).